### DISCUSSION TITLE
Somebody may have found a way to enable JIT on iOS/iPadOS 14.4+

### DISCUSSION THREAD

**Discussion by ToqQrrl** [utmapp/UTM#2475] [General]
I was checking out the Delta emulator for iOS and the developer wrote the following for his upcoming Delta 2.0 release (now on 1.3)...

https://rileytestut.com/blog/2021/04/21/delta-1-3-ds-for-everyone/

+++
That being said, there is hope! I’ve discovered a new method to enable JIT on devices running iOS 14 or later — including iOS 14.4 and the upcoming iOS 14.5 — which I plan to release in beta soon. This new method is slightly more complex than what iOS 14.2 and 14.3 support, but I plan to automate as much complexity as possible so it can “just work” — so stay tuned 😉
+++

Maybe you can get in touch with this developer ?

**Comment by ntindle** [comment]
I think that would be @rileytestut if he chose to share it

**Comment by osy** [comment]
https://trello.com/c/HjSLTOpP
